packages/server/postgres/migrations/1686226640890_addTeamHealthToAllMeetingSettingsRetrospective.ts

```diff
@@ -0,0 +1,42 @@
+import {r, RValue} from 'rethinkdb-ts'
+import {ParabolR} from '../../database/rethinkDriver'
+
+const connectRethinkDB = async () => {
+  const {hostname: host, port, pathname} = new URL(process.env.RETHINKDB_URL!)
+  await r.connectPool({
+    host,
+    port: parseInt(port, 10),
+    db: pathname.split('/')[1]
+  })
+  return r as any as ParabolR
+}
+
+export async function up() {
+  await connectRethinkDB()
+  await r
+    .table('MeetingSettings')
+    .filter({meetingType: 'retrospective'})
+    .update((row: RValue) => ({
+      phaseTypes: r.branch(
+        row('phaseTypes').contains('TEAM_HEALTH'),
+        row('phaseTypes'),
+        row('phaseTypes').contains('checkin'),
+        row('phaseTypes').insertAt(1, 'TEAM_HEALTH'),
+        row('phaseTypes').prepend('TEAM_HEALTH')
+      )
+    }))
+    .run()
```

Comment:
This needs testing on staging to see how long it will run. Needs to wait until the current release process is done.

Reply:
I've run up and down migration twice on staging with and without profiling. It takes between 45 and 60s. The load on the server was not exceptionally high, thus I think it's probably safe as is.